lib/Model/Importer/DebugInfo/PDBImporter.cpp

```diff
@@ -804,7 +804,8 @@ Error PDBImporterTypeVisitor::visitKnownRecord(CVType &Record,
       // as `static` or `friend`.
       if (Function.getMethodKind() != MethodKind::Static
           and Function.getMethodKind() != MethodKind::Friend
-          and ProcessedTypes.count(CurrentTypeIndex) != 0) {
+          and ProcessedTypes.count(CurrentTypeIndex) != 0
+          and ProcessedTypes[CurrentTypeIndex].get() != nullptr) {
         auto MaybeSize = ProcessedTypes[CurrentTypeIndex].get()->size();
         if (MaybeSize and *MaybeSize != 0) {
```

Comment:
@dwarfyresearch, ah, I almost forgot, I had to apply something like this because `model::TypePath{}.get()->size()` was failing. Not sure if this fix is sufficient though.

When you have a moment, could you, please, take a look into where these empty references come from?

Reply:
Fixed.